frontend/src/components/MessageBox.tsx

```diff
@@ -10,10 +10,10 @@ import "./styles/messageBox.css";
  */
 
 type Props = {
-  changeMessageText: Function,
-  inputText: string,
-  sendMessage: MouseEventHandler<HTMLButtonElement>
-}
+  changeMessageText: Function;
```

Comment:
In Menu.tsx for "onClick" you used a arrow function type, but here Function, is it correct place () => any here to stick one style ?

Reply:
Fixed in [Refactor type code style](https://github.com/InnoSWP/InnoSCS/pull/48/commits/79daf42a078838b94d6b70d6be2b6ee3c11e7ad8)